import "galaxy.proto"
package baidu.galaxy;

option cc_generic_services = true;

service Agent {
    rpc CreateContainer(CreateContainerRequest) returns (CreateContainerResponse);
    rpc RemoveContainer(RemoveContainerRequest) returns (RemoveContainerResponse);
    rpc ListContainers(ListContainersRequest) returns (ListContainersResponse); 
    rpc Query(QueryRequest) returns (QueryResponse);
}

